changes/5067.added

```diff
@@ -0,0 +1 @@
+Added `q` (search) filter to `ProviderFilterSet`, `ProviderNetworkFilterSet`, `ImageAttachmentFilterSet` and `ObjectPermissionFilterSet`.
```

Comment:
Let's document the new generic test cases as an addition as well.

I wonder, should we add the tests in `next` rather than `develop` just given the potential impact to apps?